actions/workflows/test_hostname.yaml

```
@@ -14,6 +14,12 @@ input:
   - dns_domain
   - hostname
 
+vars:
+  - err_message: ""
```

not necessary, but just a suggestion
```suggestion
  - error: ""
```